src/state/slices/common-selectors.ts

```diff
@@ -26,7 +26,11 @@ export const selectWalletSupportedChainIds = (state: ReduxState) =>
 export const selectWalletAccountIds = createDeepEqualOutputSelector(
   selectWalletId,
   (state: ReduxState) => state.portfolio.wallet.byId,
-  (walletId, walletById): AccountId[] => (walletId && walletById[walletId]) ?? [],
+  (state: ReduxState) => state.portfolio.disabledAccountIds,
+  (walletId, walletById, disabledAccountIds): AccountId[] => {
+    const walletAccountIds = (walletId && walletById[walletId]) ?? []
+    return walletAccountIds.filter(accountId => !disabledAccountIds.includes(accountId))
+  },
```

Comment:
This is the change needing high risk tag

Reply:
This seems sane. I like how it's fail open, i.e. if `disabledAccountIds` is empty then we'll return all `walletAccountIds`.